### ISSUE TITLE
Score not showing in correct place

### ISSUE THREAD

**Issue by hawkwind3** [Realm667/boa-addons#23]
See screenshots ...

![score1](https://github.com/Realm667/boa-addons/assets/35204218/43b8856d-c8d1-451a-a4b2-76d65f5f8169)
![score2](https://github.com/Realm667/boa-addons/assets/35204218/2f4a7089-7cc7-42e4-973e-5616f3e5f202)

I believe the score should show below the gold amount.
I have all the latest score addons from this site
Also using addon_score.boa dated 10/7/2021.

**Comment by x3003** [comment]
This is a dup of https://github.com/Realm667/boa-addons/issues/16
I sort of fixed the position of the score, but it hasn't been implemented in the git source.
So download the latest git version (also for your other probs), apply the patch manually, compile it and replace every add on w/ the new ones. In a new drawer w/ all the latest stuff ofcourse.

**Comment by hawkwind3** [comment]
I didn't realise this was a duplication.

I read #16, and those issues are still present, no not a solution.

All others I have fixed by latest git and my own changes.
________________________________

**Comment by x3003** [comment]
>those issues are still present, no not a solution.

Well, if you apply the patch, your issue should be fixed.
The only remaining thing is that the score might be obscured by the compass.
I did 3 chapters and besides this (and a side issue [too much gold]), it should work.

**Comment by hawkwind3** [comment]
I obviously did not explain myself.

I did do the patch and have downloaded and compiled the latest addons.
And yes, the position of the compass is the issue.


________________________________

**Comment by x3003** [comment]
But atleast the score is at the right position [for now]. Dunno why it changed in the first place, maybe something in gzdoom was changed [?].

**Comment by hawkwind3** [comment]
Yes. I also suspect gzdoom. Why was SetVisibility changed to IsVisible ?
